pkg/controllers/user/alert/controller.go

```diff
@@ -0,0 +1,289 @@
+package alert
```

Comment:
We should perhaps use less generic name for file name

Reply:
It is an entry point for all the alert controllers. For a specific controller like statesyncer it is easier to have specific name.  Do you have any suggestion?